src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -4643,6 +4644,33 @@ private static async Task<Process> StartProcess(string hostname, string benchmar
                 process.WaitForExit();
             };
 
+            var useWindowsLimiter = (job.MemoryLimitInBytes > 0 || job.CpuLimitRatio > 0 || !String.IsNullOrWhiteSpace(job.CpuSet)) && RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
Nit: not that it really matters, but you could do the PS check first which would save you from running the rest of the statements in case it's not Windows.

Reply:
Is there any description of what RemoteExecutor's goal is exactly. I didn't get the relationship with this after a quick look at the code.